data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Line 1: Maybe make this comment pop out more? (this might render with spaces, that isn't intended)</p><p> </p><p>######################</p><p>######comment#########</p><p>######################</p><p> </p><p>Lines 7-8: I assume this is a templating thing that gets rendered by the testing framework?</p><p> </p><p>Line 66: Blinghamton sounds like a fun place, but I believe this should be "Binghamtom".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
LOL I think you're 100% right on Binghamton what a good New Yorker catch!!

Reply:
<p>Re Lines 7-8, it does not - the user needs to fill it in, but I added a comment clarifying</p>

Reply:
<p>re lines 7-8, does a separate copy of this exist in the test environment? Only because this shouldn't be actionable by the (external) user. </p>

Reply:
<p>What does the word "this" mean in this context? </p>

Reply:
<p>Filling in the lines with their information</p>

Reply:
<p>Oops, I was not specific about which "this". I was specifically looking at the first one. So, I think you're saying that because this is just for our testing, we should not expect users to fill things in. At the top, we do tell them to skip this cell. Do you want to see this handled a different way?</p>

Reply:
<p>Sorry if I wasn't clear either. Looking at this with fresh eyes, this might be fine as is. Perhaps a comment just before saying "for test user" or something, but not 100% necessary.</p>

Reply:
<p>Done</p>